TenureInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -36,10 +36,35 @@ public static TenureResponseObject ToResponse(this TenureInformation domain)
             };
         }
 
-        public static List<TenureResponseObject> ToResponse(this IEnumerable<TenureInformation> domainList)
+        public List<TenureResponseObject> ToResponse(IEnumerable<TenureInformation> domainList)
         {
             if (null == domainList) return new List<TenureResponseObject>();
-            return domainList.Select(domain => domain.ToResponse()).ToList();
+            return domainList.Select(domain => new TenureResponseObject
+            {
+                Id = domain.Id,
+                Charges = domain.Charges,
+                EndOfTenureDate = domain.EndOfTenureDate,
+                EvictionDate = domain.EvictionDate,
+                HouseholdMembers = domain.HouseholdMembers.ToList(),
+                InformHousingBenefitsForChanges = domain.InformHousingBenefitsForChanges,
+                IsActive = domain.IsActive,
+                IsMutualExchange = domain.IsMutualExchange,
+                IsSublet = domain.IsSublet,
+                IsTenanted = domain.IsTenanted,
+                LegacyReferences = domain.LegacyReferences.ToList(),
+                Notices = domain.Notices.ToList(),
+                PaymentReference = domain.PaymentReference,
+                PotentialEndDate = domain.PotentialEndDate,
+                StartOfTenureDate = domain.StartOfTenureDate,
+                SubletEndDate = domain.SubletEndDate,
+                SuccessionDate = domain.SuccessionDate,
+                TenuredAsset = domain.TenuredAsset,
+                TenureType = domain.TenureType,
+                Terminated = domain.Terminated,
+                AgreementType = domain.AgreementType
+            }).ToList();
```

Comment:
Not sure why you changed this and replicated the entire single-object implementation here.
Couldn't you have just changed it to...
```
return domainList.Select(domain => ToResponse(domain)).ToList();
```